php/libraries/OutputWrapper.class.inc

```diff
@@ -0,0 +1,188 @@
+<?php declare(strict_types=1);
+/**
+ * This class is designed to assist in writing command-line interface tools
+ * for LORIS.
+ *
+ * It contains helper functions to format output, etc.
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class OutputWrapper
+{
+    // Colours mapped to ANSI escape codes.
+    // @see https://en.wikipedia.org/wiki/ANSI_escape_code#3/4_bit
+    const COLOURS = [
+        'red'    => '31',
+        'yellow' => '33',
+        'green'  => '32',
+    ];
+    /**
+     * Whether this script should log its output to a file or just print to
+     * STDOUT.
+     *
+     * @var bool
+     */
+    private $_loggingEnabled = false;
+    /**
+     * The directory that will contain log files written by this script.
+     *
+     * @var SPLFileInfo
+     */
+    private $_logDirectory;
+
+    /**
+     * Create a new command line helper.
+     *
+     * @param ?SPLFileInfo $logDirectory The directory that will contain log
+     *                                   files written by this script.
+     *
+     * @return void
+     */
+    public function __construct(SPLFileInfo $logDirectory = null)
+    {
+        $this->_logDirectory = new SPLFileInfo(
+            $logDirectory ?? __DIR__ . '/tools/'
```

Comment:
This path is wrong. It'll resolve to "$LORIS/php/libraries/tools"

Reply:
Good catch, thanks.